fastapi_opa/auth/auth_saml.py

```diff
@@ -39,13 +82,24 @@ async def init_saml_auth(self, request_args: Dict) -> OneLogin_Saml2_Auth:
         )
 
     @staticmethod
-    async def single_sign_on(auth: OneLogin_Saml2_Auth) -> RedirectResponse:
-        redirect_url = auth.login()
+    async def single_log_out(auth: OneLogin_Saml2_Auth) -> RedirectResponse:
+        name_id = auth.get_nameid()
```

Comment:
This data should probably be taken from the session (but I'm not sure) since you otherwise have to login to logout.

Reply:
Add deleted session before call this